### PR TITLE
Fixes #17186 - enrich docker tags ui

### PR DESCRIPTION
**Pull request by thomasmckay**

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/details/docker-tag-environments.controller.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * @requires translate
+ * @requires DockerTag
+ *
+ * @description
+ *   Provides the functionality for the docker tags details action pane.
+ */
+angular.module('Bastion.docker-tags').controller('DockerTagEnvironmentsController',
+    ['$scope', 'translate', 'DockerTag',
+    function ($scope, translate, DockerTag) {
+        if ($scope.tag) {
```
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/details/docker-tag-environments.controller.js]: In what case is this already defined and in what case is it not?

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/details/views/docker-tag-environments.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+
+    <table bst-table="table"
```
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/details/views/docker-tag-environments.html]: You could use the partial table template here if you wanted free search functionality.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/details/docker-tag-environments.controller.js (hunk 1 above)
Comment (thomasmckay) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/details/docker-tag-environments.controller.js]: I removed this controller as it was unnecessarily refetching the tag.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/details/views/docker-tag-environments.html (hunk 2 above)
Comment (thomasmckay) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/details/views/docker-tag-environments.html]: I added a filter to the table; was easier than I expected. :)

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/details/docker-tag-details.controller.js [hunk 3]
```diff
@@ -11,7 +11,7 @@
  * @description
  *   Provides the functionality for the docker tags details action pane.
  */
-angular.module('Bastion.docker-tags').controller('DockerTagsDetailsController',
+angular.module('Bastion.docker-tags').controller('DockerTagDetailsController',
```
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/details/docker-tag-details.controller.js]: Can you also update the test to reflect the name change  please?

Comment (thomasmckay) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/details/docker-tag-details.controller.js]: Write new tests you mean? The tests only cover the DockerTagsController

Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/details/docker-tag-details.controller.js]: Oh, yeah, if you don't mind wiring some for that control that would be awesome.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/details/views/docker-tag-environments.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        <a ui-sref="content-views.details.version.docker({contentViewId: tag.content_view_version.content_view_id, versionId: tag.content_view_version.id})">
+          {{ tag.content_view_version.name }}
+        </a>
+      </td>
+      <td bst-table-cell>
+        {{ tag.repository.full_path }}:{{ tag.name }}
+      </td>
+    </tr>
+  </tbody>
+</table>
```
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/details/views/docker-tag-environments.html]: This file should be using the table layout.  See my other PRs for an example.

Comment (thomasmckay) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/details/views/docker-tag-environments.html]: That layout adds a search box, which is not what this page needs. The filter I added is the style desired.

**Comment by waldenraines** [comment]
ACK

**Pull request merged**